tools/src/bin/create-image.rs

```diff
@@ -412,19 +431,23 @@ fn main() {
     }
 
     if let Some(init_paths) = matches.values_of("inis") {
-        for init_path in init_paths {
-            let program_name = std::path::Path::new(init_path);
-            process_names.set(
-                pid,
-                program_name
-                    .file_stem()
-                    .expect("program had no name")
-                    .to_str()
-                    .expect("program name is not valid utf-8"),
-            );
-            pid += 1;
-            let init = read_minielf(init_path).expect("couldn't parse init file");
-            args.add(IniS::new(init.entry_point, init.sections, init.program, init.alignment_offset));
+        if let Some(ref mut sargs) = swap_args {
```

Comment:
See if maybe this works?
```suggestion
        if let Some(sargs) = &mutswap_args {
```

Reply:
yes, that works.